install.py

```diff
@@ -76,6 +76,7 @@ def gnome_shell_version():
     'pantheon': lambda: 'pantheon',
     'unity': lambda: 'unity',
     'xfce4': lambda: 'xfce4',
+    'plasmashell': lambda: 'kde',
```

Comment:
would you mind moving this up between `pantheon` and `unity` to keep them sorted?